apps/web/scripts/installExternalCourses.py

```diff
@@ -0,0 +1,20 @@
+import json
+import sys
+import os
+
+def main():
+    if len(sys.argv) != 2:
+        exit(f"Usage: {sys.argv[0]} PATH_TO_CONFIG_FILE")
+    filename = sys.argv[1]
+    with open(filename) as fh:
```

Comment:
why `fh`? :thinking: 

It's not a big deal here as this is a pattern any Python developer will recognize without thinking, but I think abbreviations and 1-2 character variable names are generally not great. :thinking: for one I can't figure out what `fh` stands for, even though I understand the code

Reply:
I am fighting for moving from f to fh at a client, it is already quite searchable and quite acceptable, but if you'd like some other name, I open for it.

Reply:
I would just ditch abbreviations.

```python
with open(filename) as config_file:
```

Thought this is not the most critical example of abbreviations being bad